proto/scheduler.proto

```diff
@@ -206,7 +218,13 @@ message ReEnqueueTaskResponse {
 
 message EnqueueTaskReservationRequest {
   string task_id = 1;
-  TaskSize task_size = 2;
+
+  // The size of the task that determines how many resources the executor should
+  // allocate for this task.
+  //
+  // DEPRECATED: use scheduling_metadata.scheduled_size instead.
+  TaskSize task_size = 2 [deprecated = true];
```

Comment:
The comment on `SchedulingMetadata.task_size` says to use this field, and `scheduled_size` doesn't exist.

Reply:
woops, I thought I reverted this change. will revert